src/main/kotlin/uk/gov/justice/hmpps/offenderevents/services/HMPPSDomainEventsEmitter.kt

```diff
@@ -84,6 +85,7 @@ class HMPPSDomainEventsEmitter(
       "PRISONER_APPOINTMENT-UPDATE" -> PrisonerAppointmentUpdateEvent.toDomainEvents(message.fromJson())
       "IMPRISONMENT_STATUS-CHANGED" -> ImprisonmentStatusChangedEvent.toDomainEvents(message.fromJson())
       "SENTENCE_DATES-CHANGED" -> SentenceDatesChangedEvent.toDomainEvents(message.fromJson())
+      "APPOINTMENT_CHANGED" -> VideoAppointmentCancelledEvent.toDomainEvents(message.fromJson())
```

Comment:
this is slightly odd - at this stage it is just an appointment cancelled so quite confusing when it is a video appointment event

Reply:
I'll rename more appropriately - should be better as `AppointmentChangedEvent.toDomain.... `

Reply:
Renamed as a AppointmentChangeEvent.